qc_collapsed_bam/qc_collapsed_bam_v2.0__packed.cwl

```diff
@@ -3415,12 +3448,6 @@
             "requirements": [
                 {
                     "class": "SubworkflowFeatureRequirement"
-                },
```

Comment:
Why are we loosing these?, can you check the individual CWL to make sure these are part of them

Reply:
I believe this is correct: https://github.com/msk-access/cwl_subworkflows/blob/ff2358cd21fb013a08a0ad688b479f35c9b21c45/qc_collapsed_bam/qc_collapsed_bam_v2.0.cwl. There is no `InlineJavascriptRequirement` present in the qc_collapsed_bam_v2.0. The only file I actually changed was: https://github.com/msk-access/cwl_subworkflows/blob/8ab29a3fac1cc0c175a5db6cc5d6278167ff0b01/vardict_workflow/run_processed_vardict.cwl. The rest of these changes are from the pack_cwl.yaml git action. I believe these are correct changes from the action. They just aren't on develop yet, because the action didn't originally trigger due to the error in the vardict cwl.